src/test/basics/PerfLog_test.cpp

```diff
@@ -309,7 +309,9 @@ class PerfLog_test : public beast::unit_test::suite
 
         // Get the all the labels we can use for RPC interfaces without
         // causing an assert.
-        std::vector<char const*> labels{ripple::RPC::getHandlerNames()};
+        std::set<const char*> handlerNames = ripple::RPC::getHandlerNames();
+        std::vector<char const*> labels(
+            handlerNames.begin(), handlerNames.end());
```

Comment:
You're taking this `set` and converting it to a `vector` a couple of times in unit tests. The only other usage of `getHandlerNames()` is in `PerfLog`, where the collection type doesn't matter as long as it's iterable.

I was going to suggest changing the return type back to `vector`, but the overhead to copy the set is unnecessary in the non-test code. What do you think about adding a function to `TestHelpers.h/cpp`, that returns a `vector`? It could even have the same name, since it would be in the `ripple::test` namespace.

Reply:
Thanks, added a helper function in `TestHelpers.h`